### PR TITLE
Refactor the front-end to reduce template redundancy

### PR DESCRIPTION
**Pull request by bwakedavis**
- [x] placed redundant markup used by other pages in base.html
- [x] removed JavaScript and styling from html pages and placed in respective files
- [x] update the command to run the dev server in the readme which changed with the last merge

### CODE REVIEW

File: templates/about.html [hunk 1]
```diff
@@ -1,188 +1,81 @@
...
+                      <div>Spotbit can be used by anyone who wants to take advantage of its privacy, speed, reliability,
+                        and self-sovereignty advantages. It works particularly well with the <a
+                          href="https://github.com/sponsors/BlockchainCommons">Gordian system</a>, which supports Tor
+                        connections between a <a href="https://github.com/sponsors/BlockchainCommons">Gordian Wallet</a>
+                        and a <a href="https://github.com/sponsors/BlockchainCommons"></a>Gordian Server or other server
+                        installed by <a href="https://github.com/sponsors/BlockchainCommons">Bitcoin Standup
+                          scripts</a>.
+                        Spotbit is an example of a microservices meant to display how the <a
+                          href="https://github.com/BlockchainCommons/spotbit">Gordian Principles</a>, which are
```
Comment (nochiel) [inline templates/about.html:52]: Can we use the correct links:
- [Gordian System](https://github.com/BlockchainCommons/Gordian)
- [Gordian Principles](https://github.com/BlockchainCommons/Gordian#gordian-principles)
- [Gordian Wallet](https://github.com/BlockchainCommons/GordianWallet-iOS)
- [Bitcoin Standup Scripts](https://github.com/BlockchainCommons/Bitcoin-Standup-Scripts)

File: templates/about.html [hunk 2]
```diff
@@ -1,188 +1,81 @@
...
+                        philosophical and technical underpinnings to Blockchain Commons' Gordian technology, are
+                        supported by the Gordian Architecture.
+
+                        <div>Contribute to Spotbit on <a href="https://github.com/BlockchainCommons/spotbit">Github</a>
```
Comment (nochiel) [inline templates/about.html:56]: Use this url: <https://github.com/BlockchainCommons/spotbit/blob/master/CONTRIBUTING.md>

**Review by nochiel — CHANGES_REQUESTED** [review]

**Review by nochiel — APPROVED** [review]

**Comment by bwakedavis** [comment]
Thanks for the review @nochiel

**Comment by shannona** [comment]
Merging per OK by @nochiel

**Pull request merged**